settings.gradle

```diff
@@ -256,6 +256,9 @@ include ":kotlin-build-common",
         ":kotlin-scripting-compiler-embeddable",
         ":kotlin-scripting-compiler-impl",
         ":kotlin-scripting-compiler-impl-embeddable",
+        ":kotlin-scripting-ide-services-impl",
```

Comment:
why -impl, BTW? It doesn't seem like you have an interface and implementation parts?

Reply:
Renamed